batch/python/cluster_partitions.py

```diff
@@ -14,17 +14,19 @@
 @click.option("-P", "--partition_schema", type=str, help="Partition schema")
 @click.option("-x", "--index_type", type=str, help="Index type used for clustering")
 @click.option("-c", "--column_name", type=str, help="Column used for clustering")
+@click.option("-PS", "--partition_suffix", type=str, help="Column used for clustering")
```

Comment:
can't use `-PS` must be single character

Reply:
`-S` is still available

Reply:
not adding for now